scripts/run-in-new-session

```diff
@@ -0,0 +1,204 @@
+#!/usr/bin/python3
```

Comment:
This needs a copyright header and attribution to the original author (@halfline and @M4rtinK).